client/src/index.html

```diff
@@ -39,7 +39,7 @@
               <div class="header-image">
                 <a class="app-logo" id="logo">
                   <span class="logo-container">
-                    <img src="assets/icon.png" width="40" height="40">
+                    <img ng-src="{{ AppCtrl.enterprise.logo || 'assets/icon.png' }}" width="40" height="40">
```

Comment:
You can try this to see if we can get the logo automatically updating:
```html
ng-src="AppCtrl.getEnterpriseLogo()"
```

And then define the function to be:
```js
vm.getEnterpriseLogo = () => Session.enterprise && Session.enterprise.logo || 'assets/icon.png';
```

See if that works!